runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/util/HopToMode.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018, Adam <Adam@sigterm.info>
+ * Copyright (c) 2018, Kamiel
+ * Copyright (c) 2019, alanbaumgartner <https://github.com/alanbaumgartner>
+ * Copyright (c) 2019, Kyle <https://github.com/kyleeld>
+ * Copyright (c) 2019, Lucas <https://github.com/lucwousin>
```

Comment:
Thanks but I didn't write this

Reply:
Thanks for your comments ill see what I can do.